conf/prod.routes

```diff
@@ -23,6 +23,6 @@ GET       /manager/:id/accounts/:accountId/withdrawal-charges/:transactionId
 POST      /manager/:id/accounts/:accountId/transactions                             uk.gov.hmrc.lisaapi.controllers.BonusPaymentController.requestBonusPayment(id: String, accountId: String)
 GET       /manager/:id/accounts/:accountId/transactions/:transactionId              uk.gov.hmrc.lisaapi.controllers.BonusPaymentController.getBonusPayment(id: String, accountId: String, transactionId: String)
 GET       /manager/:id/accounts/:accountId/transactions/:transactionId/payments     uk.gov.hmrc.lisaapi.controllers.TransactionController.getTransaction(id: String, accountId: String, transactionId: String)
-GET       /manager/:id/payments                                                     uk.gov.hmrc.lisaapi.controllers.BulkPaymentController.getBulkPayment(id: String, startDate: String, endDate: String)
+GET       /manager/:id/payments                                                     uk.gov.hmrc.lisaapi.controllers.BulkPaymentController.getBulkPayment(id: String, startDate: Option[String], endDate: Option[String], periodStartDate: Option[String], periodEndDate: Option[String])
```

Comment:
Don't know how I feel about this. Feels wrong, but the only other approach I can think of would be to go back to having two routes files - one for v1 and one for v2. So on balance this is probably the best compromise. Interested to hear what @chrisjameswright thinks?